templates/CRM/Contribute/Form/Contribution.tpl

```diff
@@ -246,7 +240,7 @@
           {ts}Payment Details{/ts}
         </div>
         <div class="crm-accordion-body">
-        {if $contribID && $payments}
+        {if $contribID && ($payments || $isPending)}
```

Comment:
Actually this is the same as if $contribID isn't it - although I would always prefer a var like $isEditMode